frontend/src/components/facilities/FacilityInfo.vue

```diff
@@ -216,15 +201,5 @@ export default {
       return this.$route.path?.includes('/applications/')
     },
   },
-  methods: {
-    editFacility() {
-      this.setWarningAlert('This feature is not yet implemented')
```

Comment:
after we removed this setWarningAlert, I think alertMixin is unused?

Reply:
Removed.